GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -229,6 +229,7 @@ private void FormCommit_Load(object sender, EventArgs e)
             ResetUnStaged.Visible = AppSettings.ShowResetUnstagedChanges;
             CommitAndPush.Visible = AppSettings.ShowCommitAndPush;
             AdjustCommitButtonPanelHeight();
+	    showUntrackedFilesToolStripMenuItem.Checked = !Module.RunGitCmd("config status.showUntrackedFiles").Trim().Equals("no");
```

Comment:
You should use Module.EffectiveConfigFile instead and we use 4 space indentation.